woocommerce/woocommerce-gutenberg-products-block
# Product Gallery Thumbnails: Add support for cropping

## What

Fixes #11699

## Why

This PR adds support for cropping to thumbnails.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. With a block theme, open `Appearance > Editor`.
2. Open `Templates > Single Product`.
3. Select `Convert to Blocks` if you are using the Classic Templates.
4. Add the `Product Gallery` block.
5. Test with `Crop images to fit` enabled and disabled.
6. Make sure the thumbnails on the frontend get correctly cropped (or not cropped).


## Screenshots or screencast


With **Cropping** setting:

| Disabled | Enabled |
| ------ | ----- |
|<img width="537" alt="Moto_Guzzi_v7_–_newproductgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/c618dfa7-c53f-4ee1-91f8-12d7c7565262">|<img width="535" alt="Moto_Guzzi_v7_–_newproductgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/f7143ab9-8ad6-46a1-a20c-107638256d24">|

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Product Gallery Thumbnails: Add support for cropping.
